docs/monitoring-aws-lambda.asciidoc

```diff
@@ -28,6 +28,30 @@ image:images/architecture-white.png[image showing data flow from lambda function
 
 By using an AWS Lambda extension, Elastic APM agents can send data to a local Lambda extension process, and that process will forward data on to APM Server asynchronously. The Lambda extension ensures that any potential latency between the Lambda function and the APM Server instance will not cause latency in the request flow of the Lambda function itself.
 
+[[aws-lambda-overhead]]
+== Performance impact and overhead
+As described in the section on <<aws-lambda-arch>>, using Elastic APM with AWS Lambda requires to include the Elastic APM AWS Lambda extension and a corresponding Elastic APM agent to the Lambda runtime.
+These components may introduce a small overhead on the size of your function's deployment package as well as the execution duration of your function's invocations. 
+
+=== Impact on the deployment package size
+These compotents contribute a little to the uncompressed deployment package size of your Lambda function. The Elastic APM AWS Lambda extension and each of the APM agents have an uncompressed package size that is below 10MB. Thus, the overall impact of using Elastic APM on the uncompressed deployment package size of your Lambda function less than 20MB.
+
+=== Performance impact
+The <<aws-lambda-arch>> comes with the adavantage that APM data dispatching is decoupled from your function's request processing. 
```

Comment:
I'm not sure this link is necessary. This reads as:

> The [APM Architecture for AWS Lambda](https://apm-aws-lambda_279.docs-preview.app.elstc.co/guide/en/apm/guide/master/aws-lambda-arch.html) comes with the adavantage that APM data dispatching is decoupled from your function’s request processing.

Are you meaning to use the extension name instead?

 ```suggestion
 An advantage of the Elastic APM AWS Lambda extension is that APM data dispatching is decoupled from your function's request processing.
```

Reply:
Oh, I think I get it now. The advantage is because of the architecture? I still don't think we need to link to that page again, but maybe these words are better:

```suggestion
An advantage of the Elastic APM AWS Lambda extension architecture is that APM data dispatching is decoupled from your function's request processing.
```